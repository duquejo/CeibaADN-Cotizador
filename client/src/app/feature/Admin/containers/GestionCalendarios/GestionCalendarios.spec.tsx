import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import { GestionCalendarios } from './index';
import { SinonStub } from 'sinon';

describe('GestionCalendarios Test', () => {

  let componentWrapper: ShallowWrapper;
  let componentProps: React.ComponentProps<typeof GestionCalendarios> & {
    agregarNuevoCalendario:SinonStub;
    actualizarCalendario: SinonStub;
    eliminarCalendario: SinonStub;
  };

  it('DeberÃ­a renderizar el componente', () => {
    componentWrapper = shallow( <GestionCalendarios { ...componentProps } /> );
    expect(componentWrapper).toMatchSnapshot();
  });

});