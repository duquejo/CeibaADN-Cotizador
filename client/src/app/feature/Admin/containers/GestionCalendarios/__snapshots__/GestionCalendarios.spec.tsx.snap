// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GestionCalendarios Test Deber√≠a renderizar el componente 1`] = `
<Fragment>
  <h2>
    Gesti√≥n de calendarios 
    <span
      aria-label="Calendarios"
      role="img"
      title="Calendarios"
    >
      üìÖ
    </span>
  </h2>
  <p>
    Administre los calendarios que podr√° asignar a los Centros Vacacionales, los 
    <em>
      festivos o d√≠as de alta 
    </em>
    para la plataforma, corresponden evento √∫nico del d√≠a por el que cambia de manera din√°mica el precio del d√≠a completo seg√∫n el rango salarial de un usuario al momento de cotizar.
  </p>
  <styled.div>
    <styled.div>
      <CrearCalendario />
    </styled.div>
    <styled.div>
      <AccionesCalendario />
    </styled.div>
  </styled.div>
  <styled.hr />
</Fragment>
`;
